frontend/src/mocks/handlers.ts

```diff
@@ -108,12 +122,13 @@ export const defaultMocks: Mocks = {
         },
     },
     post: {
-        'https://us.i.posthog.com/e/': (): MockSignature => [200, 'ok'],
-        '/e/': (): MockSignature => [200, 'ok'],
-        'https://us.i.posthog.com/decide/': (): MockSignature => [200, 'ok'],
-        '/decide/': (): MockSignature => [200, 'ok'],
-        'https://us.i.posthog.com/engage/': (): MockSignature => [200, 'ok'],
+        'https://us.i.posthog.com/e/': (req, res, ctx): MockSignature => posthogCORSResponse(req, res, ctx),
+        '/e/': (req, res, ctx): MockSignature => posthogCORSResponse(req, res, ctx),
+        'https://us.i.posthog.com/decide/': (req, res, ctx): MockSignature => posthogCORSResponse(req, res, ctx),
+        '/decide/': (req, res, ctx): MockSignature => posthogCORSResponse(req, res, ctx),
+        'https://us.i.posthog.com/engage/': (req, res, ctx): MockSignature => posthogCORSResponse(req, res, ctx),
         '/api/projects/:team_id/insights/:insight_id/viewed/': (): MockSignature => [201, null],
+        'api/projects/:team_id/query': [200, { results: [] }],
```

Comment:
@mariusandra @Twixes is this 🌶️ 

we have a bunch of tests where we're hitting query and msw is logging a warning because there's no handler... seems nicer to just assume there's no data for incidental calls to query rather than add this line in many places